CHANGELOG.md

```diff
@@ -10,3 +10,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  - Better network error handling
  - Public members and init in CometClientHttpError and TokenProvidingHttpError
  - Added support for custom handling of 401(unauthorized) HTTP errors 
+
+### Fixed
```

Comment:
Nechceme tady začít nějaké verzování? Momentálně máme tag s 0.1.0, tak bych vytvořil klidně 0.1.1 nebo 0.2.0, dle toho, jako co to bereme a přidal do changelogu pod touto verzí.